Zilon.Core/Zilon.Core/MapGenerators/StaticObjectFactories/CherryBrushFactory.cs

```diff
@@ -12,5 +13,8 @@ public sealed class CherryBrushFactory : PropDepositFactoryBase
             IDropResolver dropResolver) : base(toolTags: Array.Empty<string>(), dropTableSchemeSid: "cherry-brush", PropContainerPurpose.CherryBrush, schemeService, dropResolver)
         {
         }
+
+        protected override int ExhausingValue { get => 1; }
+        protected override DepositMiningDifficulty DepositMiningDifficulty { get => DepositMiningDifficulty.Easy; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5398720)